### PR TITLE
[zh] Add eviction policy page

### PR DESCRIPTION
**Pull request by tengqm**
This is a sync to English site change (#23478)

### CODE REVIEW

File: content/zh/docs/concepts/scheduling-eviction/eviction-policy.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: 逐出策略
```
Comment (xichengliudui) [inline content/zh/docs/concepts/scheduling-eviction/eviction-policy.md:2]:
```suggestion
title: 驱逐策略
```

File: content/zh/docs/concepts/scheduling-eviction/eviction-policy.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+title: Eviction Policy
+content_template: templates/concept
+weight: 60
+-->
+
+<!-- overview -->
+<!--
+This page is an overview of Kubernetes' policy for eviction.
+-->
+本页提供 Kubernetes 逐出策略的概览。
```
Comment (xichengliudui) [inline content/zh/docs/concepts/scheduling-eviction/eviction-policy.md:16]:
```suggestion
本页提供 Kubernetes 驱逐策略的概览。
```

File: content/zh/docs/concepts/scheduling-eviction/eviction-policy.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+当 `kubelet` 结束一个 Pod 时，它将终止 Pod 中的所有容器，而 Pod 的 `Phase` 将变为 `Failed`。
+如果被驱逐的 Pod 由 Deployment 管理，这个 Deployment 会创建另一个 Pod 给 Kubernetes 来调度。
+
+## {{% heading "whatsnext" %}}
+
+<!--
+- Read [Configure out of resource handling](/docs/tasks/administer-cluster/out-of-resource/) to learn more about eviction signals, thresholds, and handling.
+-->
+- 阅读[配置资源不足的处理](/zh/docs/tasks/administer-cluster/out-of-resource/)，
+  进一步了解逐出信号、阈值以及处理方法。
```
Comment (xichengliudui) [inline content/zh/docs/concepts/scheduling-eviction/eviction-policy.md:43]:
```suggestion
  进一步了解驱逐信号、阈值以及处理方法。
```

File: content/zh/docs/concepts/scheduling-eviction/eviction-policy.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/concepts/scheduling-eviction/eviction-policy.md]: fine with me.

**Pull request merged**